models/project/src/mc-project/transformers.ts

```diff
@@ -0,0 +1,43 @@
+import { Transformer } from '@commercetools-test-data/core';
+import type { TMcProject, TMcProjectGraphql } from './types';
+
+const transformers = {
+  default: Transformer<TMcProject, TMcProject>('default', {
+    buildFields: [
+      'owner',
+      'expiry',
+      'suspension',
+      'allAppliedActionRights',
+      'allAppliedDataFences',
+      'allAppliedPermissions',
+      'allPermissionsForAllApplications',
+    ],
+  }),
+  rest: Transformer<TMcProject, TMcProject>('rest', {
+    buildFields: [
+      'owner',
+      'expiry',
+      'suspension',
+      'allAppliedActionRights',
+      'allAppliedDataFences',
+      'allAppliedPermissions',
+      'allPermissionsForAllApplications',
+    ],
+  }),
+  graphql: Transformer<TMcProject, TMcProjectGraphql>('graphql', {
+    buildFields: [
+      'owner',
+      'expiry',
+      'suspension',
+      'allAppliedActionRights',
+      'allAppliedDataFences',
+      'allAppliedPermissions',
+      'allPermissionsForAllApplications',
+    ],
+    addFields: () => ({
+      __typename: 'Project',
+    }),
+  }),
+};
```

Comment:
**nit**:  Since we want to have the same build fields for all transformers, we can maybe extract them to a constant so we can reuse them and make sure we always use the same ones:

```suggestion
const buildFields = [
    'owner',
    'expiry',
    'suspension',
    'allAppliedActionRights',
    'allAppliedDataFences',
    'allAppliedPermissions',
    'allPermissionsForAllApplications',
  ];

const transformers = {
  default: Transformer<TMcProject, TMcProject>('default', {
    buildFields,
  }),
  rest: Transformer<TMcProject, TMcProject>('rest', {
    buildFields,
  }),
  graphql: Transformer<TMcProject, TMcProjectGraphql>('graphql', {
    buildFields,
    addFields: () => ({
      __typename: 'Project',
    }),
  }),
};
```